### PR TITLE
adding js & html updated files

### PR DESCRIPTION
**Pull request by MattyHoughton**
pushing up javascript and html files

### CODE REVIEW

File: .vs/ProjectSettings.json
Comment (Bwalker01) [inline .vs/ProjectSettings.json:1]: i see you've added the .vs file to the .gitignore, these files shouldnt be here. something might be off with it

**Review by Bwalker01 — APPROVED** [review]
LGTM 👍

File: .vs/ProjectSettings.json
Comment (MattyHoughton) [reply .vs/ProjectSettings.json:1]: Yeah I will delete the .vs files from the PR and we can go ahead with that

**Review by michaelohanlon6 — APPROVED** [review]
POST requests LGTM

**Pull request merged**